README.md

```diff
@@ -1,4 +1,19 @@
-# Arcus Template
-Template repository for new components in Arcus ecosystem.
+# Arcus - Azure Azure Machine Learning
+[![Build Status](https://dev.azure.com/codit/Arcus/_apis/build/status/Commit%20builds/CI%20-%20Arcus.AzureML?branchName=master)](https://dev.azure.com/codit/Arcus/_build/latest?definitionId=833&branchName=master)
+[![PyPI version](https://badge.fury.io/py/arcus-azure-ml.svg)](https://badge.fury.io/py/arcus-azure-ml)
 
-:warning: In order to finalize the setup, don't forget to run out [GitHub Repo Configurator](https://github.com/arcus-azure/arcus/tree/master/tools/github-repo-configurator).
+Azure Event Grid development in a breeze.
+
+![Arcus](https://raw.githubusercontent.com/arcus-azure/arcus/master/media/arcus.png)
+
+# Installation
+
+```shell
+PM > pip3 install arcus-azure-ml
+```
+
+# Documentation
+All documentation can be found on [here](https://azureml.arcus-azure.net/).
```

Comment:
This link doesn't work, do we still need to setup docs? If so, let's leave it out for now.

Reply:
yep